### ISSUE TITLE
Effects does not work after changing Content.

### ISSUE THREAD

**Issue by mrxten** [muak/AiForms.Effects#14]
My page has different ContentViews for both orientations.
I added ripple effect for both.
After changing Content in a page to another ContentView ripple effect ceases to work.

**Comment by mrxten** [comment]
ex.
Vertical orientation - works
changed to horizontal orientation - works
changed back to vertical - does not works.

**Comment by muak** [comment]
@mrxten 
Sorry, I don't know your situation much.
Could you paste some code?

**Comment by mrxten** [comment]
Try rotate phone. After rotating vertical -> horizontal -> vertical your effects will not work in any elements.
ex. 

**Main.page.cs**
```csharp
public class Main()
{
        private readonly View _horizontalView = new HorizontalView();
        private readonly View _verticalView = new VerticalView();

        protected override void OnSizeAllocated(double width, double height)
        {
            base.OnSizeAllocated(width, height);
            Content = Width > Height ? _horizontalView : _verticalView;
        }
}
```

**VerticalView.xaml**
```xml 
<ContentView xmlns="http://xamarin.com/schemas/2014/forms" 
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:partials="clr-namespace:VisuPad2.Views.Dashboard.Partials;assembly=VisuPad2"
             xmlns:extensions="clr-namespace:VisuPad2.Extensions;assembly=VisuPad2"
             x:Class="VisuPad2.Views.Dashboard.Orientation.VerticalView">
    <Grid effects:AddCommand.On="True"
                  effects:AddCommand.EffectColor="LightGray"
                  effects:AddCommand.EnableRipple="True"
                  effects:AddCommand.Command="{Binding SomeCommand}">
    <!--something-->
    </Grid>
</ContentView>
```


**HorizontalView.xaml**
```xml
<!--something-->
```

**Comment by muak** [comment]
@mrxten 
Thank you for your code.
I turned out cause of this issue.
I Released Fixed Version 0.2.3.
Please try it.

**Comment by mrxten** [comment]
Yes, it works! Thanks.
